plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveLocalDynamicFilteringTest.java

```diff
@@ -0,0 +1,21 @@
+package io.trino.plugin.hive;
+
+import io.trino.testing.AbstractLocalDynamicFilteringTest;
+import io.trino.testing.QueryRunner;
+
+import static io.trino.tpch.TpchTable.getTables;
+
+public class TestHiveLocalDynamicFilteringTest
+        extends AbstractLocalDynamicFilteringTest {
+    @Override
+    protected QueryRunner createQueryRunner() throws Exception {
+        return HiveQueryRunner.builder()
+                // Adjust DF limits to test edge cases
+                .addExtraProperty("dynamic-filtering.small-broadcast.max-distinct-values-per-driver", "100")
+                .addExtraProperty("dynamic-filtering.small-broadcast.range-row-limit-per-driver", "100")
+                .addExtraProperty("dynamic-filtering.large-broadcast.max-distinct-values-per-driver", "100")
+                .addExtraProperty("dynamic-filtering.large-broadcast.range-row-limit-per-driver", "100000")
```

Comment:
Should these configs be contributed by AbstractLocalDynamicFilteringTest. That way all subclasses can use the constant defined there. I don't think these values would need to vary per connector.